hacker-news-app/src/Components/LogInPage/logInPage.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import { Route, Link, } from 'react-router-dom';
+import { withFormik, Form, Field, ErrorMessage } from 'formik';
+import * as Yup from 'yup';
+import axios from 'axios';
+import styled from 'styled-components';
+
+const Container = styled.div`
+    width: 400px;
+    border: 4px solid #470938;
+    margin: 100px auto;
+    display: flex;
+    flex-direction: column;
+    border-radius: 10px;
+    text-align: center;
+`;
+
+const H1 = styled.h1`
+    width: 101%;
```

Comment:
`101%`? For real? 😅

Reply:
Styling is still incomplete. 100% wasn't working, it was a hacky fix. This is still to be sorted.